.github/workflows/deploy.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy
+
+on:
+  pull_request_target:
+    branches: 
+      - main
+    types: [ closed ]
```

Comment:
Shouldn't we deploy on any push to `main`, even if they did not originate from a merged pull request? 🤔 
This would ensure a consistent mapping between commits on `main` and deployments, and simplify the code by removing the need to check `if: github.event.pull_request.merged == true`.

If changed, this should be reflected in the CHANGELOG.